Dockerfile

```diff
@@ -1,26 +1,23 @@
 # ---------- Builder Stage ----------
-# Explicitly specify platform for ARM Mac compatibility
 FROM --platform=linux/amd64 rocker/shiny:latest AS builder
 
-# Install system dependencies needed for building packages
 RUN apt-get update && apt-get install -y --no-install-recommends \
     git \
     libcurl4-openssl-dev \
     libssl-dev \
     libxml2-dev \
     && rm -rf /var/lib/apt/lists/*
 
-# Set CRAN repository to use precompiled binaries
-ENV CRAN_REPO="https://packagemanager.posit.co/cran/latest"
+ENV CRAN_REPO="https://packagemanager.posit.co/cran/2025-04-22"
+
+ENV NEMOGB_PKG="nemogb-dev/nemogb-r@main"
```

Comment:
Similarly to the R packages, should this be tagged to a release? Otherwise we can't look at the dockerfile and know what version of the app its running.